doc/reference/configuration/configuration_reference.rst

```diff
@@ -2174,3 +2174,340 @@ The ``security`` section defines configuration parameters related to various sec
     | Type: boolean
     | Default: false
     | Environment variable: TT_SECURITY_SECURE_ERASING
+
+..  _configuration_reference_snapshot:
+
+snapshot
+--------
+
+The ``snapshot`` section defines configuration parameters related to the :ref:`snapshot files <internals-snapshot>`.
+To learn more about the snapshots' configuration, check the :ref:`Persistence <configuration_persistence_snapshot>` page.
+
+..  NOTE::
+
+    ``snapshot`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`snapshot.dir <configuration_reference_snapshot_dir>`
+-   :ref:`snapshot.snap_io_rate_limit <configuration_reference_snapshot_snap_io_rate_limit>`
+-   :ref:`snapshot.count <configuration_reference_snapshot_count>`
+-   :ref:`snapshot.by.* <configuration_reference_snapshot_by>`
+
+    -   :ref:`snapshot.by.interval <configuration_reference_snapshot_by_interval>`
+    -   :ref:`snapshot.by.wal_size <configuration_reference_snapshot_by_wal_size>`
+
+..  _configuration_reference_snapshot_dir:
+
+..  confval:: snapshot.dir
+
+    A directory where memtx stores snapshot (.snap) files. Can be relative to
+    ``process.work_dir``. If not specified, defaults to
```

Comment:
Nit: 'can be' feels strange in a reference guide.

Any relative path in this option (including the defualt value) is interpreted as relative to `process.work_dir`.